internal/worker/sshsession/worker.go

```diff
@@ -0,0 +1,363 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package sshsession
+
+import (
+	"bufio"
+	"context"
+	"io"
+	"net"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/juju/errors"
+	"github.com/juju/worker/v3"
+	"github.com/juju/worker/v3/catacomb"
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/juju/juju/core/watcher"
+	"github.com/juju/juju/internal/worker/authenticationworker"
+	"github.com/juju/juju/rpc/params"
+)
+
+var ControllerSSHUser = "reverse-ssh"
```

Comment:
Use the constant from the `internal/sshtunneler/` package instead. If it's not exported, export it.